client/components/data/sync-reader-follows/index.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External Dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import { getReaderFollowsLastSyncTime } from 'state/selectors';
+import { requestFollows } from 'state/reader/follows/actions';
+
+const TIME_BETWEEN_SYNCS = 1000 * 60 * 60; // one hour
+
+class SyncReaderFollows extends Component {
+	check() {
```

Comment:
we could move the logic up to a selector, sure. that would make it easier to test.